samples/client/petstore/swift5/frozenEnums/PetstoreClient/Classes/OpenAPIs/Models/EnumTest.swift

```diff
@@ -16,49 +16,23 @@ public struct EnumTest: Codable, Hashable {
         case upper = "UPPER"
         case lower = "lower"
         case empty = ""
-        // This case is a catch-all generated by OpenAPI such that the enum is "non-frozen".
-        // If new enum cases are added that are unknown to the spec/client, they are safely
-        // decoded to this case. The raw value of this case is a dummy value that attempts
-        // to avoids collisions with previously specified cases.
-        case unknownDefaultOpenAPI = "unknown_default_open_api"
+        case unknownDefaultOpenApi = "unknown_default_open_api"
     }
     public enum EnumStringRequired: String, Codable, CaseIterable, CaseIterableDefaultsLast {
         case upper = "UPPER"
         case lower = "lower"
         case empty = ""
-        // This case is a catch-all generated by OpenAPI such that the enum is "non-frozen".
-        // If new enum cases are added that are unknown to the spec/client, they are safely
-        // decoded to this case. The raw value of this case is a dummy value that attempts
-        // to avoids collisions with previously specified cases.
-        case unknownDefaultOpenAPI = "unknown_default_open_api"
+        case unknownDefaultOpenApi = "unknown_default_open_api"
     }
     public enum EnumInteger: Int, Codable, CaseIterable, CaseIterableDefaultsLast {
         case _1 = 1
         case number1 = -1
-        // This case is a catch-all generated by OpenAPI such that the enum is "non-frozen".
-        // If new enum cases are added that are unknown to the spec/client, they are safely
-        // decoded to this case. The raw value of this case is a dummy value that attempts
-        // to avoids collisions with previously specified cases.
-        //
-        // 192, used to calculate the raw value, was the Swift Evolution proposal for
-        // frozen/non-frozen enums.
-        // [SE-0192](https://github.com/apple/swift-evolution/blob/master/proposals/0192-non-exhaustive-enums.md)
-        //
-        case unknownDefaultOpenAPI = -11184809 // Int.min / 192
+        case numberUnknownDefaultOpenApi = -11184809
```

Comment:
@jarrodparkes Here we have a side effect that it add's the prefix number to the enum case name... I'm not sure if it's worth the effort to fix this... What do you think?

Reply:
is keeping the case name `unknownDefaultOpenAPI` across all enums possible? if so, id say it is desirable. im not sure of the amount of work required to keep it consistent

Reply:
I don't know either, but I will investigate 👍

Reply:
Done 👍